docs/recipes/rspress.mdx

```diff
@@ -0,0 +1,86 @@
+# Rspress
+
+:::info Prerequisites
+
+- Finished our [Get Started](/general/get-started) guide.
+- Registered an account on [Zephyr Cloud](https://app.zephyr-cloud.io).
+- Installed npm >=8.
+- Installed node >=18.
+
+If you don't have `nvm` to manage your node versions yet, head to [the official nvm guide](https://github.com/nvm-sh/nvm) to install it.
+
+```bash
+nvm use 18
+```
+
+## Generating a Rspress project
+
+```bash filename="Terminal"
+npm create rspress@latest my-rspress-site
+```
+
+and then run commands to make sure you have necessary dependencies installed.
+
+```bash filename="Terminal"
+cd my-rspress-site
+npm install
+```
+
+## Add Zephyr configuration
+
+You should see the website running on localhost. Open `rspress.config.ts` in your editor and add the following modifications:
+
+```ts filename="vite.config.ts"
+import path from 'node:path';
+import { defineConfig } from 'rspress/config';
+import { withZephyr } from 'zephyr-rspress-plugin'; // Add this line
+
+export default defineConfig({
+  ...
+  ssg: true,
+  builderPlugins: [],
+  plugins: [withZephyr()], // Add this line
+});
+```
+
+:::warning
+
+**Remember to initialize a git repository in current folder**, before you `npm run build` to deploy your app.
+
+:::
+
+With git config initialized, configured, and linked to remote origin, you are ready to deploy:
+
+```bash filename="Terminal"
+npm run build
+```
+
+You should see something similar to the following build output below. The highlighted lines include your deployed URL:
+
+```bash {16-18}
+$ rspress-ssg@1.0.0 build
+$ rspress build
+
+🔥 Rspress v1.43.12
+
+[zephyr] ready   Built in 1.73 s (web)
+ready   Built in 1.36 s (node)
+File (web)                                    Size        Gzip
+doc_build/static/js/styles.f3d5533f.js        0.17 kB     0.13 kB
+doc_build/index.html                          1.1 kB      0.57 kB
+doc_build/static/js/async/537.c8f80cb6.js     1.9 kB      1.0 kB
+doc_build/static/css/styles.e45a8283.css      55.0 kB     12.0 kB
+Total: 13.7 kB (gzip: 142.3 kB)
+[zephyr] ✓ You are now logged in to Zephyr Cloud
+[zephyr] rspress-ssg.zephyr-packages.zephyrcloudio#280
+[zephyr] Uploaded ci snapshot in 855ms
+[zephyr] (4/4 assets uploaded in 1191ms, 20.61kb)
+[zephyr] Deployed to Zephyr's edge in 3454ms.
+[zephyr] https://domain.zephyr-cloud.io
+```
+
+:::warning
+
+\*\*The total size for the (web) thread represents the original files but the uploaded total size means the SSG files already processed. If the SSG flag is disabled the Rspress plugin relies in the Rspack plugin execution.
```

Comment:
Put in an observation that "If ssg is enabled", otherwise I think it really represents the output right?

Reply:
I think I phrased it badly.
Put in an observation that says that the if the ssg is turned off, the bundle will be what the output is.